server-spi-private/src/main/java/org/keycloak/models/AccountRoles.java

```diff
@@ -26,6 +26,7 @@
     String MANAGE_ACCOUNT = "manage-account";
     String INITIATE_ACTION = "initiate-action";
     String MANAGE_ACCOUNT_LINKS = "manage-account-links";
+    String MANAGE_APPLICATAIONS = "manage-applications";
```

Comment:
@douglaspalmer still have to review the rest of the code. But there's a typo above.

Reply:
@douglaspalmer thinking more about this now. Do we think we need any changes into our migrations?

Reply:
@abstractj would it be better to require MANAGE_ACCOUNT instead?